vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_util_unittest.cc

```diff
@@ -80,8 +80,7 @@ TEST_F(UpholdUtilTest, GetFeeAddress) {
 TEST_F(UpholdUtilTest, GetAuthorizeUrl) {
   // production
   ledger::_environment = type::Environment::PRODUCTION;
-  std::string result =
-      uphold::GetAuthorizeUrl("rdfdsfsdfsdf", true);
+  std::string result = uphold::GetAuthorizeUrl("rdfdsfsdfsdf", true);
```

Comment:
Just FYI: we've only been reformatting the parts of the code that we've been changing, to help make the reviews more manageable (the linter will ignore unmodified lines).

Reply:
Right, totally makes sense!